fishtank/src/cluster.ts

```diff
@@ -10,6 +10,11 @@ import { Node } from './node'
 
 export const DEFAULT_IMAGE = 'ironfish:latest'
 export const CLUSTER_LABEL = 'fishtank.cluster'
+export const DEFAULT_CONTAINER_DATADIR = '/root/.ironfish'
```

Comment:
Nit: the prefix "DEFAULT" suggests that this can be customized, but it's not the case

Reply:
updated